.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,62 @@
+name: github pages
```

Comment:
I don't think we need this action at all because we're not publishing live docs for this project.

Reply:
Deleted!